### DISCUSSION TITLE
Help needed with storage access on QField 2

### DISCUSSION THREAD

**Discussion by lidarlady** [opengisch/QField#2866] [Q&A]
I have been happily working in the field with QField on my tablet for some years. But now I want to set up a new project from QGIS to QField and find it's all changed. I am unable to transfer as before. I don't want to work via cloud, just simple as before. Can anyone help please ...totally stuck on how to package my project ready for QField

**Comment by nirvn** [comment]
@lidarlady , I highly recommend you take the time to read the storage access on Android documentation we published recently (https://docs.qfield.org/get-started/storage/), you will likely find the answer you need, and along the way learn about new ways to utilize QField :)

More specifically, I think what you are looking for is addressed here (https://docs.qfield.org/get-started/storage/#accessing-imported-content-directly-through-usb-cable), whereas you should copy your prepared project into a subfolder of <drive>:/Android/data/ch.opengis.qfield/files/ of your device.

**Reply by ntrip6** [reply]
Thank you for this reply it was interesting to learn about the new security measures from Google restricting file access. I too have struggled to find data for import through the various Import buttons. 
The USB "cable" link in QField is helpful but I think it would be good to support read-only access for two common use cases:
1- packages and files received via email attachment (a shortcut to Downloads folder perhaps)
2- packages and files downloaded from the web browser.

**Comment by lidarlady** [comment]
Many thanks for your help Nirvn. I have done as you suggested and am pleased to report that I now have my new project onto the tablet ready for going out later this week

**Comment by nirvn** [comment]
@ntrip6 , Google's new storage restrictions also affect read-only access. The policy aims at protecting people's data, hence why it covers read-only access too (wouldn't be very efficient otherwise :) )

To answer your two points:
1/QField now does a very good job at  opening datasets directly from email apps, you shouldn't have any problem opening datasets from gmail (or compressed projects into a ZIP file)
2/ Similarly, it can open files from the browser directly
